### PR TITLE
chore(dev-site): fix dev site spacing and styling issues for the left pane and header

### PR DESCRIPTION
**Pull request by chrisdholt**
<body>

<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-development-site-react/src/components/shell/pane.tsx [hunk 1]
```diff
@@ -27,8 +27,7 @@ const style: ComponentStyles<IShellPaneManagedClasses, IDevSiteDesignSystem> = {
         width: toPx(40),
     },
     shellPane__expanded: {
-        width: toPx(300),
-        padding: toPx(12)
+        width: toPx(300)
```
Comment (nalogiudice) [inline packages/fast-development-site-react/src/components/shell/pane.tsx]: Removing the padding would affect the form generator configuration style... is this accounted for?

Comment (chrisdholt) [reply packages/fast-development-site-react/src/components/shell/pane.tsx]: The form generator is using the same style...ok. I'll figure out a way to address this singular instance.

Comment (nalogiudice) [reply packages/fast-development-site-react/src/components/shell/pane.tsx]: Another option would be to bake this style into the form generator surrounding div or form element

Comment (chrisdholt) [reply packages/fast-development-site-react/src/components/shell/pane.tsx]: Proposal is up. Added an optional prop for className to enable styling within a layout so it doesn't have to be baked into the form itself.

Comment (nalogiudice) [reply packages/fast-development-site-react/src/components/shell/pane.tsx]: Cool looks good.

**Pull request merged**